collect_scripts.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright 2021 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import shutil
+import sys
+from pathlib import Path
+from stat import S_IREAD, S_IWUSR, S_IWGRP, S_IRGRP, S_IEXEC
+
+FILE_EXTENSIONS = [".py", ".sh", ".json", ".pl", ".obo"]
+
+IGNORE_EXTENSIONS = [
+    ".cwl",
+]
+
+IGNORE_LIST = [
+    "__init__.py",
+    "__init__",
+    "copy_utils.py",
```

Comment:
We need to review all scripts in /hps/nobackup2/production/metagenomics/pipeline/tools-v5/utils folder. Many of them are outdated and useless. We can do it after migration (just remove from CODON)

Reply:
That is the idea of this script, we will just collect the scripts from the pipeline on deployment. Any changes on that folder will be discarded on each new Jenkins release.